src/Reflection/ReflectionType.php

```diff
@@ -86,7 +86,7 @@ public function __toString()
                 return 'bool';
             case $this->type instanceof Types\Float_:
                 return 'float';
-            case $this->type instanceof Types\Void:
+            case $this->type instanceof Types\Void_:
```

Comment:
OK, maybe it would be better to remove the `case`? And maybe others that are handled by the `default` branch?